### DISCUSSION TITLE
Segmentation fault

### DISCUSSION THREAD

**Discussion by ohsOllila** [TinkerTools/tinker-gpu#179] [Q&A] [answered]
Dear Tinker9 community,

I have managed to install tinker9 on my computer, but it gives segmentation fault when I try to use it. 
Can you help me to figure out if the issue is with my installation or with some incompatible versions of compilers etc.?

Output of tinker9 info:

 Primary GPU package :  CUDA

 GPU Device :  Setting Device ID to 0 from GPU utilization

 Program Information

    Version:                             1.0.0 GIT cdbd05e3
    Synchronized with Tinker commit:     5aa9948d
    C++ compiler:                        nvc++ 21.11.0
    Size of real (bytes):                4
    Size of mixed (bytes):               8
    Using deterministic force:           true
    Debug mode:                          off
    Platform:                            CUDA and OpenACC
    Primary GPU package:                 CUDA
    Latest CUDA supported by driver:     11.2
    CUDA runtime version:                11.5
    Thrust version:                      1.9.3 patch 0
    CUDA compiler:                       nvcc 10.0.130
    OpenACC compiler:                    nvc++ 21.11.0
    GPU detected:                        1
    GPU 0:                              
       PCI:                              0000:01:00.0
       Name:                             GeForce RTX 2080
       Maximum compute capability:       7.5
       Single double perf. ratio:        32
       Compute mode:                     default
       Error-correcting code (ECC):      off
       Clock rate (kHz):                 1710000
       Number of Multiprocessors:        46
       Number of CUDA cores:             5888
       Used/Total GPU memory:            14.38 % / 7.79 GB
Segmentation fault (core dumped)

and output of an attempt to run the example run:

tinker9 dynamic dhfr2.xyz 1000 1 10 1

     ######################################################################    
   ##########################################################################  
  ###                                                                      ### 
 ###            Tinker9  --  Software Tools for Molecular Design            ###
 ##                                                                          ##
 ##                      Version 1.0.0-rc  January 2021                      ##
 ##                                                                          ##
 ##                 Copyright (c)  Zhi Wang & the Ponder Lab                 ##
 ###                           All Rights Reserved                          ###
  ###                                                                      ### 
   ##########################################################################  
     ######################################################################    

 Compiled at:  22:49:12  Dec 30 2021
 Commit Date:  Thu Dec 16 04:40:53 2021 -0600
 Commit:       cdbd05e3

 ################################################################
 Joint Amber-CHARMM Benchmark on Dihydrofolate Reductase in Water
 23558 Atoms, 62.23 Ang Cube, 9 Ang Nonbond Cutoffs, 64x64x64 PME
 ################################################################
 Primary GPU package :  CUDA

 GPU Device :  Setting Device ID to 0 from GPU utilization
 Molecular Dynamics Trajectory via r-RESPA MTS Algorithm
Segmentation fault (core dumped)

BR,
Samuli Ollila
Academy research fellow
Institute of Biotechnology
University of Helsinki
https://www2.helsinki.fi/en/researchgroups/biophysical-chemistry

**Comment by zhi-wang** [comment, marked as answer]
From the `tinker9 info` output, I found these lines interesting:

```
Latest CUDA supported by driver:     11.2
CUDA runtime version:                11.5
```

This means for some reason, you are trying to use Cuda 11.5 runtime but the Cuda driver (which is installed separately to your system) is not up-to-date.

**Reply by ohsOllila** [reply]
Thanks a lot. I updated the drivers and it works now.
